PowerEditor/src/WinControls/shortcut/shortcut.rc

```diff
@@ -19,49 +19,49 @@
+    CONTROL         "&ALT",IDC_ALT_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,12,48,28,10
+    CONTROL         "&SHIFT",IDC_SHIFT_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,57,39,35,10
+    COMBOBOX        IDC_KEY_COMBO,110,37,64,270,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
+    PUSHBUTTON      "OK",IDOK,39,66,50,14
+    PUSHBUTTON      "Cancel",IDCANCEL,95,66,50,14
+    LTEXT           "+",IDC_STATIC,45,39,8,8
+    LTEXT           "+",IDC_STATIC,96,39,8,8
+    LTEXT           "This will disable the accelerator",IDC_WARNING_STATIC,6,84,170,8
+    LTEXT           "CONFLICT FOUND!",IDC_CONFLICT_STATIC,6,84,170,8
 END
 
-IDD_SHORTCUTSCINT_DLG DIALOGEX 0, 0, 286, 114
+IDD_SHORTCUTSCINT_DLG DIALOGEX 0, 0, 282, 115
 STYLE DS_SETFONT | DS_FIXEDSYS | WS_POPUP | WS_CAPTION | WS_SYSMENU
 EXSTYLE WS_EX_DLGMODALFRAME | WS_EX_WINDOWEDGE
 CAPTION "Shortcut"
 FONT 8, TEXT("MS Shell Dlg"), 400, 0, 0x1
 BEGIN
-    CONTROL         "CTRL",IDC_CTRL_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,114,27,33,10
-    CONTROL         "ALT",IDC_ALT_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,114,48,28,10
-    CONTROL         "SHIFT",IDC_SHIFT_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,159,39,35,10
+    RTEXT           "&Name:",IDC_NAME_STATIC,117,9,34,8
+    EDITTEXT        IDC_NAME_EDIT,158,8,118,12,ES_AUTOHSCROLL | ES_READONLY
+    CONTROL         "&CTRL",IDC_CTRL_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,114,27,33,10
+    CONTROL         "&ALT",IDC_ALT_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,114,48,28,10
+    CONTROL         "&SHIFT",IDC_SHIFT_CHECK,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,159,39,35,10
     COMBOBOX        IDC_KEY_COMBO,212,37,64,270,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-	PUSHBUTTON      "Apply", IDC_BUTTON_APPLY, 210, 66, 48, 14
-    PUSHBUTTON      "OK",IDOK,144,84,48,14
-    PUSHBUTTON      "Cancel",IDCANCEL,210,84,48,14
+    PUSHBUTTON      "Appl&y",IDC_BUTTON_APPLY,197,66,50,14
+    PUSHBUTTON      "OK",IDOK,141,84,50,14
+    PUSHBUTTON      "Cancel",IDCANCEL,197,84,50,14
     LTEXT           "+",IDC_STATIC,147,39,8,8
     LTEXT           "+",IDC_STATIC,198,39,8,8
-    EDITTEXT        IDC_NAME_EDIT,158,6,118,14,ES_AUTOHSCROLL | ES_READONLY
-    LTEXT           "Name :",IDC_NAME_STATIC,117,9,34,8,0,WS_EX_RIGHT
-    LTEXT           "This will remove shortcut from this command",IDC_WARNING_STATIC,6,101,162,8
-    LTEXT           "CONFLICT FOUND!",IDC_CONFLICT_STATIC,6,101,162,8
+    LTEXT           "This will remove shortcut from this command",IDC_WARNING_STATIC,6,102,162,8
+    LTEXT           "CONFLICT FOUND!",IDC_CONFLICT_STATIC,6,102,162,8
     LISTBOX         IDC_LIST_KEYS,6,6,90,72,LBS_NOINTEGRALHEIGHT | WS_VSCROLL | WS_TABSTOP
-    PUSHBUTTON      "Add",IDC_BUTTON_ADD,6,84,42,14
-    PUSHBUTTON      "Remove",IDC_BUTTON_RMVE,56,84,40,14
-END
+    PUSHBUTTON      "A&dd",IDC_BUTTON_ADD,6,84,42,14
+    PUSHBUTTON      "Remo&ve",IDC_BUTTON_RMVE,54,84,42,14
+END
```

Comment:
@alankilborn 
could you restore newline at EOF

Reply:
>could you restore newline at EOF

Done...I think.